src/containers/Profile/ProfileBounties.module.scss

```diff
@@ -11,6 +25,11 @@
   flex-direction: column;
 }
 
+.bodyInnerContainer {
+  max-width: 1040px;
```

Comment:
So no projector support @corwinharrell :)

Reply:
derp I meant to change this to `rem`